### DISCUSSION TITLE
Team C - Course 3

### DISCUSSION THREAD

**Discussion by fregataa** [lablup/contribution-academy-2021#62] [Show and tell]
follow the [link](https://github.com/lablup/contribution-academy-2021/wiki/Course-3:-%EA%B0%9C%EB%B0%9C-%ED%99%98%EA%B2%BD-%EC%84%A4%EC%A0%95%ED%95%98%EA%B8%B0) and upload the result.

**Comment by fregataa** [comment]
1. `pyenv exec backend.ai run -c 'print("hello world!")' python`
![image](https://user-images.githubusercontent.com/44239739/130316933-95f3d7c7-8473-4f68-b837-10cce635deff.png)

2. `pyenv exec backend.ai run --rm --exec "python ./single_mnist.py" tf2 ./mnist.py ./single_mnist.py`
![image](https://user-images.githubusercontent.com/44239739/130317326-8d2142da-1d3d-4a14-8c8b-a018cf31a0a4.png)

**Reply by eunjungchoi** [reply]
혹시 tf2 이미지를  어떻게 어디서 pull 받으셨나요? CLI로 하셨나요?

**Comment by jinseo-ok** [comment]
1. `pyenv exec backend.ai run -c 'print("hello world!")' python`
![image](https://user-images.githubusercontent.com/48677363/130365137-396abdc9-336f-41f1-9216-729586729e8c.png)

2. `backend.ai run --rm --exec 'python single_mnist.py' tf2 ./single_mnist.py ./mnist.py`
![image](https://user-images.githubusercontent.com/48677363/130365150-41c18475-cc4b-4b07-a578-36d6fa9df953.png)
